src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -556,12 +621,27 @@ void DlgPrefSound::masterLatencyChanged(double latency) {
     update();
 }
 
-void DlgPrefSound::headDelayChanged(double value) {
+void DlgPrefSound::masterDelaySpinboxChanged(double value) {
+    m_pMasterDelay->set(value);
+}
+
+void DlgPrefSound::headDelaySpinboxChanged(double value) {
     m_pHeadDelay->set(value);
 }
 
-void DlgPrefSound::masterDelayChanged(double value) {
-    m_pMasterDelay->set(value);
+void DlgPrefSound::boothDelaySpinboxChanged(double value) {
+    m_pBoothDelay->set(value);
+}
+
+void DlgPrefSound::roundTripLatencySpinboxChanged(double value) {
+    m_pRoundTripLatency->set(value);
+    roundTripLatencyChanged(value);
+}
+
+void DlgPrefSound::roundTripLatencyChanged(double newRoundTripLatency) {
+    double inputLatencyCompensation = newRoundTripLatency / 2.0;
```

Comment:
The input latency is what needs to be compensated for, not the round trip latency.